notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,2260 @@
+{
```

Comment:
<p>You can say that LevelSet is the implementation of the definition 2) Local Implicit Function from before.</p><p> </p><p>NIT: real-valued function f (not f(x), f(x) is a number and not a function)</p><p> </p><p>TYPO: fucntion</p><p> </p><p>--&gt; a level set is a set of points for which the function $f$ takes the exact same value. This value is called the "level", and does not need to be a scalar, it could be a vectorial value too.</p><p> </p><p>--&gt; A manifold can be defined as the pre-image of that constant value (level) by a submersion f: R^N -&gt; R^N-d, for example, if the level is the vector 0, then a manifold can be defined as f^{-1}({0}) for a given submersion f.</p><p> </p><p>I am not sure that I understand this: "When you are considering a transformation operation, the pre-image is the original appearance of a shape before the transformation."</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>We wouldn't call f a "level set transformation", it would be called a map or a submersion.</p>

Reply:
<p>I took out the weird submersion definition and just left it at (this is like definition 2 that we saw above). also, i changed the image in the way i THOUGHT you were suggesting to change the image, but please give me more feedback if what i did is not what you meant!</p>